### PR TITLE
[WEB-880] clinic signup and administration

### PR DESCRIPTION
**Pull request by cbwebdevelopment**
See [WEB-880]

Related PRs
---
https://github.com/tidepool-org/tideline/pull/426
https://github.com/tidepool-org/platform-client/pull/124

[WEB-880]: https://tidepool.atlassian.net/browse/WEB-880

### CODE REVIEW

**Review by krystophv — DISMISSED** [review]
Edit: I read a bot comment and thought this was ready for review when it's still in progress, my bad.

**Review by krystophv — CHANGES_REQUESTED** [review]
Excellent (and huge) work overall. Couple of small edits to make.

File: test/unit/components/navbar/NavigationMenu.test.js [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
+          payload: {
+            args: ['/workspaces'],
+            method: 'push',
+          },
+        },
+      ]);
+
+      // Click personal workspace option
+      store.clearActions();
+      menuOptions.at(2).simulate('click');
```
[the remark this answers is not in the document]
Comment (cbwebdevelopment) [reply test/unit/components/navbar/NavigationMenu.test.js:287]: Yep. Removing.

File: test/unit/pages/patients.test.js [hunk 2]
```diff
@@ -475,5 +534,68 @@ describe('Patients', () => {
...
+    //     expect(result.patients).to.deep.equal([
+    //       {
+    //         userid: 'd4e5f6',
+    //       },
+    //       {
+    //         userid: 'x1y2z3',
+    //       },
+    //     ]);
+    //   });
+    // });
```
[the remark this answers is not in the document]
Comment (cbwebdevelopment) [reply test/unit/pages/patients.test.js:599]: Removing this. This was for when the plan was to re-use the patients view as part of the clinic workspace.

**Comment by cbwebdevelopment** [comment]
@krystophv All feedback has been addressed

**Review by krystophv — APPROVED** [review]
LGTM 🚀

**Pull request merged**